code/game/gamemodes/cult/talisman.dm

```diff
@@ -33,56 +17,56 @@ Rite of Disorientation
 	if(!iscultist(user))
 		user << "<span class='danger'>There are indecipherable images scrawled on the paper in what looks to be... <i>blood?</i></span>"
 		return
-	if(invocation)
+	if(invoke(user))
 		user.whisper(invocation)
```

Comment:
This line is unneeded; lines 30-31 do it and this'll runtime if for some reason it has no invocation.